pkg/kubelet/metrics/metrics.go

```diff
@@ -938,6 +943,30 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		},
 	)
+
+	// DRAOperationsDuration tracks the duration of the DRA PrepareResources and UnprepareResources requests.
+	DRAOperationsDuration = metrics.NewHistogramVec(
+		&metrics.HistogramOpts{
+			Subsystem:      DRASubsystem,
+			Name:           DRAOperationsDurationKey,
+			Help:           "Latency histogram in seconds for the duration of handling all ResourceClaims referenced by a pod when the pod starts or stops. Identified by the name of the operation (PrepareResources or UnprepareResources) and separated by the success of the operation. The number of failed operations is provided through the histogram's overall count.",
+			Buckets:        metrics.DefBuckets,
```

Comment:
DefBuckets = https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#DefBuckets

> Most likely, however, you will be required to define buckets customized to your use case. 

Have you considered this? I'm asking because I don't see it mentioned and this could simply be copy-and-pasted.

The defaults range from 0.005 seconds to 10 seconds. We allow individual gRPC calls to run up to 45 seconds (https://github.com/kubernetes/kubernetes/blob/e15d5b9da913a2e7e7d190a8aa1d6c577872f015/pkg/kubelet/cm/dra/plugin/registration.go#L44), which is quite a bit longer. But perhaps we really expect calls to be faster and thus don't care if a few stragglers all go into the 10s bucket? OTOH, if we have many operations at 15s and only a few with very long times, we cannot distinguish those.

I suppose we can keep it as-is for now. We can still change it because [changing bucket boundaries is not a breaking change](https://github.com/kubernetes/kubernetes/issues/127384#issuecomment-2432163068) (but discouraged).

Reply:
Yes, I used DefBuckets as I don't have a data to define custom ones yet. The idea was to change it when we have more data.

Reply:
Then please create an issue and link to it here.

Reply:
Will do after this PR is merged, so I can refer to the merged code in the issue.

Reply:
The usual practice is to link TODOs or remarks in the code to issues that then need to get created first. Then reviewers can be sure that creating that issue won't be forgotten. You could create the issue, link to the PR, and then later update it to link to the code.

But this isn't a blocker.

Reply:
I've decided to add a custom buckets based on this discussion. Please, take a look at the last commit, thanks.